build/AuthorizationServerInterface.php.html

```diff
@@ -0,0 +1,199 @@
+<!DOCTYPE html>
```

Comment:
add to .ignore and remove from PR, also better rebase the branch, so it's not part of the history.